charts/restate-helm/replicated-values.yaml

```diff
@@ -0,0 +1,18 @@
+replicaCount: 3
+
+env:
+  - name: POD_IP
+    valueFrom:
+      fieldRef:
+        fieldPath: "status.podIP"
+  - name: RESTATE_ADVERTISED_ADDRESS
+    value: "http://$(POD_IP):5122"
+  - name: RESTATE_ROLES
+    value: '["metadata-server", "admin", "worker", "log-server"]'
+  - name: RESTATE_METADATA_CLIENT__ADDRESSES
+    value: '["http://restate:5122"]'
```

Comment:
I think this should only become a problem if the cluster get extremely large because then it might take quite a bit until you hit restate-0 by chance (assuming that's the node that was provisioned).

Reply:
Will I get on any dns resolution attempt a new pod ip? Can it be a problem if the result gets cached?

Reply:
the pods will come up one at a time, until the pod before you is ready (ie, has reached another node to bootstrap), you will not start. so i dont think this is an issue
its not done at dns level - that dns name resovles to a single 'virtual ip', and each tcp conn to that ip reaches a new pod. no caching problem